public/sass/components/_buttons.scss

```diff
@@ -221,6 +221,58 @@ $btn-service-icon-width: 35px;
   }
 }
 
+//Toggle button
+
+.toggle-btn {
```

Comment:
this class seems to use a bunch of hard coded colors, how will it look in dark theme? and the left / right things seems unnecessary (also not folloiwing BEM class naming http://getbem.com/naming/) if it's a modifier it should be double dash (`toggle-btn--left`). But left and right sub modifiers should not be needed as you can use first-child and last-child css pseudo classes. I would have a container (wrapper) called "toggle-buttons"  and each item "togge-buttons__item"

Reply:
I seem to have rushed this pr to get to the next thing. Will fix.